subtests/docker_cli/run/run_tmpfs.py

```diff
@@ -0,0 +1,28 @@
+from run import run_base
+
+
+class run_tmpfs(run_base):
+    # Verify behavior when multiple --name options passed
+
+    def init_subargs(self):
+        super(run_tmpfs, self).init_subargs()
+        self.sub_stuff['expected_option'] = \
+            self.config['run_options_csv'].split('--tmpfs ')[1]
+
+    def run_once(self):
+        super(run_tmpfs, self).run_once()
+        cid = self.sub_stuff['cid'] = self.sub_stuff['dkrcmd'].stdout.strip()
+        self.sub_stuff['containers'].append(cid)
+        try:
+            self.sub_stuff["cont"].wait_by_name(self.sub_stuff['name'])
+        except ValueError:
+            pass  # container already finished and exited
+
+    def postprocess(self):
+        super(run_tmpfs, self).postprocess()
+        cont = self.sub_stuff["cont"]
+        json = cont.json_by_long_id(self.sub_stuff['cid'])
+        self.failif(len(json) == 0)
+        actual_option = str(json[0]['HostConfig']['Tmpfs']).split("'")[1]
```

Comment:
This is fragile: it depends on an exact string representation. One fairly easy way to fix it might be to compare dicts directly:

    expected_dict = { self.config['tmpfs_path'] : '' }
    actual_dict = json[0]['HostConfig']['Tmpfs']
    self.failif_ne(actual_dict, expected_dict, 'Container .HostConfig.Tmpfs info setting')